### PR TITLE
Improving Step 8

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2216

To conform with current steps https://support.google.com/a/answer/7378726

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Additional migration limitations are described in the following table:
 
    ![Unique ID](../media/gsuite-mig-6-unique-id.png)
 
-8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, enable the checkbox for **Enable G Suite Domain-wide Delegation** and click **Save** at the bottom of the page.
+8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Then, click the Edit button at the top of the screen and then enable the checkbox for **Enable G Suite Domain-wide Delegation** and click **Save** at the bottom of the page.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:106]:
```suggestion
8. If you see an area that says **Show Domain-Wide Delegation**, click to expand that section. Click the **Edit** button at the top of the screen and enable the checkbox to **Enable G Suite Domain-wide Delegation**, and then click **Save** at the bottom of the page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**